lava-master/scripts/setup.sh

```diff
@@ -20,10 +20,15 @@ if [ -e /root/lava-users ];then
 		if [ $SUPERUSER -eq 1 ];then
 			USER_OPTION="$USER_OPTION --superuser"
 		fi
-		echo "Adding username $USER DEBUG(with $TOKEN / $PASSWORD / $USER_OPTION)"
-		lava-server manage users add --passwd $PASSWORD $USER_OPTION $USER || exit 1
-		if [ ! -z "$TOKEN" ];then
-			lava-server manage tokens add --user $USER --secret $TOKEN || exit 1
+		lava-server manage users list | grep -q "[[:space:]]$USER$"
+		if [ $? -eq 0 ];then
+			echo "Skip $USER DEBUG(with $TOKEN / $PASSWORD / $USER_OPTION)"
```

Comment:
This message should be more useful than "Skip".  It appears that it's skipping because the user already exists?  If so, the message should be more clear.

Also the changelog is not descriptive enough on this patch.  It mentions the setup.sh stuff, but nothing about these changes to user management.

Reply:
I have added a more explicit message and updated commit message